server/index.js

```diff
@@ -12,6 +12,7 @@ app.get('/house_images', (req,res) => {
    console.log("Request query:", id);
 
```

Comment:
You're missing a semicolon on lines 1, 4, 5, 6.. Consider taking a few minutes to make sure you're using either the Hack Reactor or Airbnb style guides :)